src/HashBus.Twitter.Monitor/App.cs

```diff
@@ -1,38 +1,43 @@
 namespace HashBus.Twitter.Monitor
 {
     using System.Threading.Tasks;
+    using HashBus.Application.Events;
     using HashBus.NServiceBusConfiguration;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static async Task RunAsync(
-            string nserviceBusConnectionString,
-            string track,
-            string consumerKey,
-            string consumerSecret,
-            string accessToken,
-            string accessTokenSecret,
-            string endpointName)
+        public static async Task RunAsync(string nserviceBusConnectionString, string track, string consumerKey, string consumerSecret, string accessToken, string accessTokenSecret, string endpointName, string appicationAddress)
         {
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
-            busConfiguration.ApplyMessageConventions();
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (var bus = Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();

```

Comment:
Please insert a blank line before this line (for consistency with other endpoints).